### PR TITLE
ci: fix codecov upload

### PR DESCRIPTION
**Pull request by phbelitz**
## Description

Fixes codecov upload.

## Checklist

- [x] PR is rebased to/aimed at branch `develop`
- [x] PR follows [Contributing Guide](https://github.com/sse-secure-systems/connaisseur/blob/master/docs/CONTRIBUTING.md)
- [x] Added tests (if necessary)
- [x] Extended README/Documentation (if necessary)
- [x] Adjusted versions of image and Helm chart in `values.yaml` and `Chart.yaml` (if necessary)

### CODE REVIEW

File: .github/workflows/.reusable-unit-test.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ jobs:
         uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
       - name: Install
         run: |
+          apt update && apt install -y git
```
Comment (xopham) [inline .github/workflows/.reusable-unit-test.yml:18]: you should be able to use the non-slim python container above and not have to do an update/install here, which may be faster

Comment (xopham) [reply .github/workflows/.reusable-unit-test.yml:18]: also, I do believe that "ubuntu-latest" does contain python, so I am not certain why we chose the container to begin with.

Comment (phbelitz) [reply .github/workflows/.reusable-unit-test.yml:18]: tried the `ubuntu:22.04` container ... doesn't have python or git installed ... :(

**Review by xopham — APPROVED** [review]
lgtm

**Pull request merged**